openc3-cosmos-cmd-tlm-api/app/controllers/model_controller.rb

```diff
@@ -39,6 +42,9 @@ def create(update_model = false)
       OpenC3::Logger.info("#{@model_class.name} created: #{params[:json]}", scope: params[:scope], user: username())
     end
     head :ok
+  rescue StandardError => error
+    render json: { status: 'error', message: error.message }, status: 500
+    OpenC3::Logger.error(error.formatted)
```

Comment:
Logger errors cause messages in CmdTlmServer - Probably don't want that here.   Change to using rails logger.